### ISSUE TITLE
quiet should control whether converge is reported by Chef

### ISSUE THREAD

**Issue by mhedgpeth** [chef-cookbooks/audit#65]
### Cookbook version
0.12

### Chef-client version
12.8.1
### Platform Details
Windows 2012 R2

### Scenario:
When I run `chef-client` it always reports that a resource was converged, even when `node['audit']['quiet'] = true`. 

### Steps to Reproduce:
Set `node['audit']['interval']['enabled'] = true`
Run `chef-client` twice

### Expected Result:
0 resources converge

### Actual Result:
1 resource converges (the compliance report)

```
Recipe: audit::default
  * directory[D:/chef/cache/compliance] action create (up to date)
  * file[D:/chef/cache/compliance/windows] action nothing (skipped due to action :nothing)
  * compliance_profile[windows] action fetch (skipped due to only_if)
  * compliance_profile[windows] action execute (skipped due to only_if)
  * compliance_report[chef-server] action execute
    - report compliance profiles' results
←[0m
Running handlers:
Running handlers complete
←[0m
Chef Client finished, 1/84 resources updated in 01 minutes 03 seconds
```

The `compliance_report` resource in this case is reporting a converge.

**Comment by mhedgpeth** [comment]
Looking at the code, it looks like it's written incorrectly. The quiet attribute only controls what goes into the `chef-client` log. That's not what the variable should be for; it should control whether resources report as converging in chef-client.

I'm not well-versed with this style of resource to know how to pull that off. If you provide direction on how to make the `Chef::Resource` NOT report a converge, I'd be happy to make the change in a PR.

**Comment by chris-rock** [comment]
This is solved with with version 2.0 of this cookbook. We work on the last details, but we use your proposed solution with Chef handler
